packages/notification-center/src/components/notification-center-component.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { FunctionComponent } from 'react';
+import { IMessage, MessageActionStatusEnum, ButtonTypeEnum } from '@novu/shared';
+
+import { INovuProviderProps, NovuProvider } from './novu-provider';
+import { IPopoverNotificationCenterProps, PopoverNotificationCenter } from './popover-notification-center';
+import { NotificationBell } from './notification-bell';
+import { useNotifications } from '../hooks';
+import { reactToWebComponent } from '../utils';
+import { ColorScheme } from '../shared/config/colors';
+
+type NovuProviderProps = Pick<
+  INovuProviderProps,
+  | 'onLoad'
+  | 'stores'
+  | 'backendUrl'
+  | 'socketUrl'
+  | 'subscriberId'
+  | 'applicationIdentifier'
+  | 'subscriberHash'
+  | 'i18n'
+> & {
+  // Angular props
+  sessionLoaded?: INovuProviderProps['onLoad'];
+};
+type PopoverProps = Pick<
+  IPopoverNotificationCenterProps,
+  | 'onUrlChange'
+  | 'onUnseenCountChanged'
+  | 'onActionClick'
+  | 'onTabClick'
+  | 'theme'
+  | 'tabs'
+  | 'showUserPreferences'
+  | 'offset'
+  | 'position'
+> & {
+  onNotificationClick?: IPopoverNotificationCenterProps['onNotificationClick'];
+  colorScheme?: ColorScheme;
+  // Angular props
+  urlChanged?: IPopoverNotificationCenterProps['onUrlChange'];
+  notificationClicked?: IPopoverNotificationCenterProps['onNotificationClick'];
+  unseenCountChanged?: IPopoverNotificationCenterProps['onUnseenCountChanged'];
+  actionClicked?: IPopoverNotificationCenterProps['onActionClick'];
+  tabClicked?: IPopoverNotificationCenterProps['onTabClick'];
+};
+type BellProps = { unseenBadgeColor?: string; unseenBadgeBackgroundColor?: string };
+
+export type NotificationCenterComponentProps = NovuProviderProps & PopoverProps & BellProps;
```

Comment:
the props on this component are a combination of what the three components do provide... in the next PRs we also want to improve the way these components can be styled